src/main/java/org/terasology/simpleFarming/systems/BushAuthoritySystem.java

```diff
@@ -115,6 +119,11 @@ public void onPlantDestroyed(DoDestroyPlant event, EntityRef entity, BushDefinit
         } else {
             onBudDestroyed(bushComponent, event.isParentDead);
         }
+        EntityRef selected = localPlayer.getCharacterEntity().getComponent(CharacterHeldItemComponent.class).selectedItem;
+        boolean b = selected.getParentPrefab().getName().equalsIgnoreCase("SimpleFarming:gloves");
```

Comment:
I think this condition should either be inside the if statement (`if(selected.getParentPrefab().`...`) {`) (my preference), or it should have a more descriptive name like `glovesUsed`.